### PR TITLE
Update readme and add more arch supported

### PR DESCRIPTION
**Pull request by bookshiyi**

### CODE REVIEW

**Comment by knopp** [comment]
Are the 32bit windows and linux targets actually working?

**Comment by knopp** [comment]
Linux ArmV7 is still open https://github.com/flutter/flutter/issues/75823. I haven't tested linux/x86 does it work?

**Comment by bookshiyi** [comment]
@knopp 

Oh, At first, I just wanted this project to support more architectures, but I didn't notice the problem of flutter.

I'm trying to develop a CI workflow for this repository now.

and [Run-On-Arch](https://github.com/marketplace/actions/run-on-architecture) integrated maybe a good choice for us to test cross-platform compilation.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+## Characteristic
+
+| Target      | Toolchains | Type              | Suffix | armv7        | arm64 | x86           | x86_64 |
+| ----------- | ---------- | ----------------- | ------ | ------------ | ----- | ------------- | ------ |
+| **Android** | Gradle     | C Dynamic Library | .so    | ✅           | ✅    | ✅            | ✅     |
+| **iOS**     | Cocoapods  | Static Library    | .a     | ✅           | ✅    | -             | ✅     |
+| **macOS**   | Cocoapods  | Static Library    | .a     | ✅           | ✅    | -             | ✅     |
+| **Linux**   | cmake      | C Dynamic Library | .so    | ✅(Untested) | ✅    | ✅ (Untested) | ✅     |
+| **Windows** | cmake      | C Dynamic Library | .dll   | -            | -     | ✅(Untested)  | ✅     |
```
Comment (bookshiyi) [inline README.md:25]: How do you think about this?

**Pull request closed without merging**